crates/transaction-pool/src/validate/eth.rs

```diff
@@ -185,6 +188,11 @@ impl EthTransactionValidatorBuilder {
         self.eip1559 = eip1559;
         self
     }
+    /// Sets the local transaction propagation toggle
```

Comment:
This needs a bit more context:

basically if set to true (default), all locally received transactions (via RPC for example) will be propagated over the network

Reply:
makes sense, added a bit more context